rust/cargo-psibase/src/main.rs

```diff
@@ -71,6 +64,15 @@ struct Args {
 
 #[derive(Parser, Debug)]
 struct DeployCommand {
+    /// API Endpoint URL (ie https://psibase-api.example.com) or a Host Alias (ie prod, dev). See `psibase config --help` for more details.
```

Comment:
Comment should say `cargo psibase config`, since there is actually a different CLI tool just called "psibase".